src/styles/_catalog.scss

```diff
@@ -1,22 +1,42 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  align-items: center;
+  justify-content: center;
   margin: 0 auto;
+  gap: 48px;
+  max-width: 944px;
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
+
+@media (max-width: 488px) {
+  .catalog {
+    grid-template-columns: $card-width;
+  }
+}
```

Comment:
write @media inside .catalog selector, use nesting

Reply:
Yes, you are right, it's better! Corrected +